bindings/wasm/src/crypto/key_collection.rs

```diff
@@ -133,7 +133,7 @@ impl KeyCollection {
 
     let iter: _ = data.keys.iter().flat_map(|data| {
       let pk: PublicKey = decode_b58(&data.public).ok()?.into();
-      let sk: SecretKey = decode_b58(&data.secret).ok()?.into();
+      let sk: PrivateKey = decode_b58(&data.private).ok()?.into();
 
       Some((pk, sk))
```

Comment:
Probably worth using `public_key/private_key` or `public/private` variable names here too (GitHub doesn't allow posting a suggestion on "deleted lines" unfortunately).

I imagine the `pk/pk` naming conflict is why people started using `Secret` (`sk`) instead.

Reply:
Resolved locally.